sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/AppConfigurationPropertySource.java

```diff
@@ -89,7 +94,7 @@ public final class AppConfigurationPropertySource extends EnumerablePropertySour
 
     private final HashMap<String, KeyVaultClient> keyVaultClients;
 
-    private final ClientStore clients;
+    private final ConfigurationClientWrapper clientWrapper;
```

Comment:
The name of `ConfigurationClientWrapper` is too general.

Reply:
How does `AppConfigurationReplicaClient` sound?

Reply:
Much better.